extra/release_tool.py

```diff
@@ -3087,10 +3107,10 @@ def main():
         "--in-integration-version",
         dest="in_integration_version",
         metavar="VERSION",
-        help="Used together with the above argument to query for a version of a "
+        help="Used together with the `-g` argument to query for a version of a "
```

Comment:
Suggestion:
Use the long option in the help text?
:)

Reply:
It's pre-existing, but worth a cleanup anyway, enjoy https://github.com/mendersoftware/integration/pull/1836/commits/95ad4e735d036e20bbb88d8afe00bfc2841ef263!